dev/local/Makefile

```
@@ -101,6 +101,37 @@ web:
 		--network delphi-net --name delphi_web_epidata \
 		delphi_web_epidata >$(LOG_WEB) 2>&1 &
 
+.PHONY=web_debug
+web_debug:
```

```suggestion
.PHONY=web_vscodedebug
web_vscodedebug:
```
rename so its clear that this is specifically for vscode